server/src/test/java/org/opensearch/index/codec/CodecTests.java

```diff
@@ -115,15 +115,15 @@ public void testExceptionCodecNull() {
     }
 
     // write some docs with it, inspect .si to see this was the used compression
-    private void assertStoredFieldsCompressionEquals(Lucene95Codec.Mode expected, Codec actual) throws Exception {
-        SegmentReader sr = getSegmentReader(actual);
+    private void assertStoredFieldsCompressionEquals(Lucene95Codec.Mode expected, Codec codec) throws Exception {
```

Comment:
This can still called as actual, the previous comment was specifically for ```getSegmentReader``` method.